tests/unit_tests/tables.fixtures.js

```diff
@@ -82,4 +82,283 @@ tables.buttons_event_action = [
 ];
 
 
+// =================
+// Converting from old table to new table
+// =================
+tables.original_formatting = [
```

Comment:
The amount of repetition in this test setup makes me slightly nervous that copy+paste errors while writing the test could result in spurious test failures. Is it possible to write tests where the input is just the original table (as the developer would write it) and it gets compared to the processed version of the new-style table (again, as the developer would write it)?

Reply:
That's a complex question. Created #188 for it.